packages/@okta/vuepress-site/guides/sign-into-spa/section01/index.md

```diff
@@ -2,25 +2,28 @@
 title: Before You Begin
 ---
 
-This guide walks you through how to sign users into your single-page app (SPA) and supports apps created using:
-
-* Angular
-* React
-* Vue
+This guide will show you how to use Okta as the user store for your single-page application, and sign users into your app.
 
 <!-- If you are building a web app that is served by a server framework, see [Sign Users into Your Web App]. If you are building a mobile app, see [Sign Users into Your Mobile App]. -->
 
-## Before You Begin
-
-These instructions assume that you: 
+It assumes that you:
 
-* Have an Okta Developer organization. If you don't, you can get one here: <https://developer.okta.com/signup>
-* Have a project or application that you want to add authentication to.
+* Have an Okta organization. If you don't, you can get one here: <https://developer.okta.com/signup>
+* Know the basics of building JavaScript applications.
+* Have an application you want to add authentication to. If you don't, follow the instructions below.
 
-If you don't have an app created or are new to creating an app, start with the appropriate documentation. It'll walk you through the creation of an app, creating routes, and other application development essentials:
+Throughout the guide, you'll see sections like this that are tailored to the language or framework you're building in:
 
 <StackSelector snippet="create-app"/>
 
-> Note: If you'd prefer to download a complete sample application, you can use ours:
+## Understand the redirect flow
+
+Here's how signing a user into a single-page app works:
+
+1. The user clicks a button in your app to sign in, or attempts to access a protected resource.
+1. Your application redirects the user's browser to a sign-in page hosted by Okta.
+1. The user signs in using a username and password, or other factors.
+1. Okta redirects the browser back to your callback route with tokens for the user. Your code (or our SDK's code) handles the callback and stores these tokens.
+1. The browser is redirected to the final destination. This is up to you: it can be the route the user was originally trying to access, the home page, or somewhere else.
 
-<StackSelector snippet="samples"/>
+<button>I'm ready to add auth to my app</button>
```

Comment:
@alexdahl-okta Can you work w/ @swiftone to define the CSS for these buttons? (Or links that look like buttons, if that's what they should be)

Reply:
Will do! (Indeed, likely a link that looks like a button.)

@swiftone The magenta "Create Free Account" buttons on [code pages](https://developer.okta.com/code/javascript/) may be a good model (`class="Button--red"`).